ui/nuxeo-slots.js

```diff
@@ -148,7 +158,15 @@ window.nuxeo.slots.setSharedModel = (model) => {
 
     disconnectedCallback() {
       super.disconnectedCallback();
-      this._unregister(this.slot);
+      this._unregister(this.name || this.slot);
```

Comment:
nit: would it make sense to have a `slotName` getter instead of doing the `this.name || this.slot` everywhere?

Reply:
It would. Will do, thanks.